src/OpenTelemetry/Metrics/MeterProviderSdk.cs

```diff
@@ -125,6 +125,15 @@ internal sealed class MeterProviderSdk : MeterProvider
                         var metricStreamConfig = viewConfig(instrument);
                         if (metricStreamConfig != null)
                         {
+                            if (metricStreamConfig is HistogramConfiguration histogramConfiguration)
+                            {
+                                // Validate the histogram bounds
+                                if (!this.IsSortedAndDistinct(histogramConfiguration.BucketBounds))
+                                {
+                                    throw new ArgumentException("Must be in ascending order with distinct values", $"{histogramConfiguration}.{histogramConfiguration.BucketBounds}");
```

Comment:
we shouldn't throw here. as this code is occuring after SDK has successfully initialized.